### PR TITLE
feat: wallet UI

### PR DESCRIPTION
**Pull request by BalogunofAfrica**
Fixes https://github.com/bitcointranscripts/transcription-review-front-end/issues/141

### CODE REVIEW

File: src/pages/wallet/index.tsx [hunk 1]
```diff
@@ -1,129 +1,243 @@
...
-          <strong>Wallet</strong>
-          <Text>Balance: {walletData.balance}</Text>
-          <Text>id: {walletData.id}</Text>
+        <Box mb="10">
+          <Heading size={"md"} mb={6}>
+            My Wallet
+          </Heading>
+          <Flex gap={3} flexDir="column">
+            <Flex gap={2}>
+              <Text fontWeight={"bold"}>ID:</Text>
```
Comment (Extheoisah) [inline src/pages/wallet/index.tsx:156]: we don't want to show the wallet id. the user doesn't need this info

File: src/pages/wallet/index.tsx [hunk 2]
```diff
@@ -1,129 +1,243 @@
...
+              </Flex>
+            </Flex>
+            <Flex alignItems={"center"} gap={2}>
+              <Text fontWeight={"bold"}>Balance: </Text>
+              {isLoading ? (
+                <Spinner color="orange.500" size={"xs"} />
+              ) : (
+                <Text fontWeight={"medium"}>{walletData?.balance ?? 0}</Text>
+              )}
+              <Button
```
Comment (Extheoisah) [inline src/pages/wallet/index.tsx:186]: We should disable this button if the balance is zero (0)

**Review by Extheoisah — CHANGES_REQUESTED** [review]

**Comment by Extheoisah** [comment]
We should also sort the transactions date from newest to oldest

File: src/components/alerts/WalletAlert.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    if (invoiceInput === "") {
+      return;
+    }
+    payInvoice.mutate(
+      {
+        invoice: invoiceInput,
+        userId: sessionDataId,
+      },
+      {
+        onSuccess: (response) => {
```
Comment (Extheoisah) [inline src/components/alerts/WalletAlert.tsx:55]: on success, we should close the withdraw dialog

**Comment by BalogunofAfrica** [comment]
@Extheoisah I believe the sorting should be done from the query point, i.e the backend

File: src/pages/wallet/index.tsx [hunk 4]
```diff
@@ -1,129 +1,202 @@
...
+        <Heading size={"md"} mb={10}>
+          My Wallet
+        </Heading>
+        <Flex mb={6} alignItems={"center"} gap={2}>
+          <Text fontWeight={"bold"}>Balance: </Text>
+          {isLoading ? (
+            <Spinner color="orange.500" size={"xs"} />
           ) : (
-            <p>No transactions</p>
+            <Text fontWeight={"medium"}>{walletData?.balance ?? 0}</Text>
```
Comment (Extheoisah) [inline src/pages/wallet/index.tsx:144]: lets show the currency denomination e.g. 21,000 SAT

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**